src/components/organisms/UiMultipleAnswer/UiMultipleAnswer.vue

```diff
@@ -51,6 +51,7 @@
           name="list-item"
         >
           <UiMultipleAnswerItem
+            :ref="(el)=>{ if (el) setFirstMultipleAnswerItemRef(el, index) }"
```

Comment:
I've got this idea that just popped into my head, and it _may_ be a solution to this _ugly `if`_ statement in the function. What if we just `setFirstMultipleAnswerItemRef(el as InstanceType<typeof UiMultipleAnswerItem>, index)`? Then the function **_maybe_** could be typed like:

```
const setFirstMultipleAnswerItemRef = (
  el: InstanceType<typeof UiMultipleAnswerItem>,
  index: number,
) => {
  if (index === 0) firstMultipleAnswerItemRef.value = el.content.input;
};
```
Can we check it?